packages/actions/index.ts

```diff
@@ -715,6 +715,18 @@ export const logoutAccount = () => {
   }
 }
 
+export const removeAccount = () => {
+  return async (dispatch, getState) => {
+    const { address, privateKeyBuffer } = getUser(getState())()
+    await mnemonicStorage.remove()
+    await hashedPasswordStorage.remove()
+    await creditProtocol.deleteChannelID(address, privateKeyBuffer)
+    const payload = { hasStoredUser: false, user: undefined }
+    dispatch(displaySuccess(accountManagement.logoutSuccess))
+    dispatch(setState(payload))
```

Comment:
Careful here. `creditProtocol.deleteChannelID()` can easily fail (e.g: device is offline, server was unavailable for some reason, some other server/network issue). When that happens, `removeAccount()` would fail and the app would find itself in a bad state.

I'm less concerned about the storage removals failing, although I suppose that, too, could go awry.

To mitigate this issue, we could try deleting the channel ID first (i.e: the one most likely to fail), then if that succeeds, continue on to doing the other things. If something fails, we need a path for the app to recover.